### DISCUSSION TITLE
Specmatic GraphQL/GRPC with c# and docker

### DISCUSSION THREAD

**Discussion by smitakolhe** [specmatic/specmatic#1205] [Ask For Help]
I want to do a POC with Garphql/grpc using specmatic. Where can I download the TRIAL version of specmatic for that?
Also I couldn't find how to use docker commands for the same in the documentation. Could someone please help with this?

Thank you.
@harikrishnan83 @joelrosario  Could you please help?

**Comment by harikrishnan83** [comment]
@smitakolhe Thanks for reaching out. We will get back to you shortly on this.
